tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3394,6 +3391,21 @@ std::tuple<int, int> GetTensorVectorSizeAndDim(
   return std::make_tuple(vector_size, vector_dim);
 }
 
+cudnn_frontend::Tensor CreateCudnnTensor(
+    const int64_t* dims, const int64_t* strides, int64_t ndims, int64_t uid,
```

Comment:
absl::Span<const int64_t> would be safer and more conventional for these arrays (and would make call sites more convenient, too).